theRAPTLab/meme-2023
# dev-tv/fix-comment-warnings: Fixes #5 warning on new comment

# Summary
This PR fixes #5  where several errors regarding invalid `className` appeared in the console when clicking on the Comment button

---

## User-visible changes:

Console no longer fills up with errors/warnings after clicking on the Comment button

---

## Developer-relevant changes:

The fix relies on wrapping the elements with the `styled` property before rendering them, solution found in [this commit](https://github.com/theRAPTLab/meme-2023/commit/42d38f5620660143174b24f9fba714fe93db5f18)

---

## Testing Steps
1. Pull the changes from the branch dev-tv/fix-comment-warnings
2. Run the application.
3. Create a new comment.
4. Verify that clicking on the new comment button does not fill the console with errors / warnings

---

## Technical Notes
Other than the described fix, we also changed the property `rowsMax` on the  MUI `<Input />` component to be `maxRows`, as that is the correct syntax for it in [MUI v5](https://mui.com/material-ui/api/input/)
